README.md

```diff
@@ -14,7 +14,8 @@ MaterialX is an open standard for representing rich material and look-developmen
 ### Quick Start for Developers
 
 - Download the latest version of the [CMake](https://cmake.org/) build system.
-- Point CMake to the root of the MaterialX library and generate C++ projects for your platform and compiler.
+- Create a new sub-folder called `build` under the root of the MaterialX library.
```

Comment:
I like the idea of clarifying our start-up process, but by adding a manual step where the user creates a `build` subfolder, we're effectively steering the user towards command-line usage of CMake and out-of-source builds, since this step is not required from the CMake GUI or for in-source builds.

Maybe we should leave our Quick Start as it stands, but provide details for a few different CMake scenarios in our full Developer Guide?

I'm definitely open to thoughts on this, and I'm now so used to CMake (both on the command-line and from the GUI) that I can no longer be objective about which instructions are valuable.

Reply:
Sounds good to not clutter this up and leave it in the Developer Guide.

I think for the Developer Guide, I propose something like this instead:

"It is recommended to create a separate build folder, with the default folder name of `build` to match the default Github setup.`, then the blurb about running the build from that folder.

BTW, I found that on some CMake configs that it errors out without using `-S ..` vs `..` for the source directory.